parl/core/torch/agent.py

```diff
@@ -147,3 +150,18 @@ def restore(self, save_path, model=None, map_location=None):
             model = self.alg.model
         checkpoint = torch.load(save_path, map_location=map_location)
         model.load_state_dict(checkpoint)
+
+    def train(self):
+        """Sets the agent in training mode.
+
+        This has any effect only on certain modules of their behaviors in training/evaluation mode, if they are affected, e.g. class:`Dropout`, class:`BatchNorm`, etc.
```

Comment:
This has any effect only on certain modules, which behave differently in train/evaluation mode, eg. `Dropout`, `BatchNorm`, etc.